client/me/purchases/manage-purchase/index.jsx

```diff
@@ -414,7 +414,7 @@ const ManagePurchase = React.createClass( {
 
 		if ( isIncludedWithPlan( purchase ) ) {
 			const attachedPlanUrl = paths.managePurchase(
-				this.props.selectedSite.domain,
+				this.props.domain,
```

Comment:
`this.props.domain` isn't defined here. I believe should be `this.props.selectedSite.slug`.

There is a bug here though: this block will never render if the purchase is expiring. I'll add a commit to address both of these issues.